go.mod

```diff
@@ -1,6 +1,8 @@
 module terraform-provider-genesyscloud
 
-go 1.20
+go 1.21
+
+toolchain go1.22.3
```

Comment:
You'll have to undo this. `toolchain` causes problems with the tool we use to release, so we have to keep the go verison the same. Sometimes this update to go.mod happens automatically based on the go version you have installed and the value of the env var GOROOT